telethon/_client/messages.py

```diff
@@ -738,3 +738,41 @@ async def _pin(self, entity, message, *, unpin, notify=False, pm_oneside=False):
 
     # Pinning a message that doesn't exist would RPC-error earlier
     return self._get_response_message(request, result, entity)
+
+async def send_reaction(
+    self: 'TelegramClient',
+    entity: 'hints.EntityLike',
+    message: 'hints.MessageIDLike',
+    reaction: typing.Optional[str] = None
+):
+    message = utils.get_message_id(message) or 0
+    if not reaction:
+        get_default_request = _tl.fn.help.GetAppConfig()
+        app_config = await self(get_default_request)
+        reaction = (
+            next(
+                (
+                    y for y in app_config.value
+                    if "reactions_default" in y.key
+                )
+            )
+        ).value.value
+    request = _tl.fn.messages.SendReaction(
+        peer=entity,
+        msg_id=message,
+        reaction=reaction
+    )
+    result = await self(request)
+    for update in result.updates:
+        if isinstance(update, _tl.UpdateMessageReactions):
+            return update.reactions
+    return reaction
```

Comment:
Is there any case in which this can happen? Perhaps we should construct a `messageReactions` instance manually. I wouldn't like methods to have strange return values for v2 which then must be kept forever.

Reply:
should a new reactions type be defined inside https://github.com/LonamiWebs/Telethon/tree/master/telethon/types/_custom ?

Reply:
> in a basic groups, and private chats, currently we are getting a UpdateEditMessage type.

In my opinion, we should always return a `messageReactions` instance. I don't think it's worth it to return a custom object, since this is mostly informational.

Reply:
i am not sure if there is a case, when Telegram won't return anything. 

so far tested by adding reaction in private chats, basic groups, supergroups, and channels.